packages/js/manifests/polywrap/src/formats/polywrap.codegen/0.1.0.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export interface CodegenManifest {
+  /**
+   * Polywrap codegen manifest format version.
+   */
+  format: "0.1.0" | "0.1";
+  /**
+   * Output directory for the generated code (default: ./src/wrap)
+   */
+  codegenDir?: string;
```

Comment:
Camel case isn't use in any of our other schemas, AFAIK we've been using snake case (see `import_abis`).

Also I think we should work on the naming of these properties to be a bit more ergonomic and grouped by functionality. For example, you could group `codegenDir` and `publishDir` into a single root property which is `output`.

For example, these could be valid:
```yaml
output:
  types: ...
  manifest: ...
```

The existing naming is pretty bad :P sorry about that